client/src/components/user/dashboard.js

```diff
@@ -1,39 +1,265 @@
+      ? <div className="p-3 m-2 d-flex flex-wrap borderSubscriptionForm">
+          <div className="col d-flex justify-content-between align-items-center">
+            <div />
+            <h2 className="text-start">Create Subscription</h2>
+            <button className="btn btn-link my-2 d-md-none" type="button" data-bs-dismiss="offcanvas" aria-label="Close" onClick={() => this.setState({ addingSubscription: !addingSubscription })}>
+              <FontAwesomeIcon icon={faTimes} />
+            </button>
+            <button className="btn btn-link my-2 d-none d-md-block" type="button" onClick={() => this.setState({ addingSubscription: !addingSubscription })}>
+              <FontAwesomeIcon icon={faTimes} />
+            </button>
+          </div>
+          <CreateSubscription 
+          addSubscription={this.handleUpdate}
+          toggleLoadingState={this.toggleLoadingState}
+          showSubscriptionList={this.showSubscriptionList}
+          currentSubscriptions={subscriptions} />
+        </div>
+      : <div className="p-3 m-2 d-flex flex-wrap borderSubscriptionForm">
+          <div className="col d-flex justify-content-between align-items-center">
+            <div />
+            <h2 className="text-start">Update Subscription</h2>
+            <button className="btn btn-link my-2 d-md-none" type="button" data-bs-dismiss="offcanvas" aria-label="Close" onClick={() => this.setState({ editingSubscription: !editingSubscription, activeSubscription: false })} >
+              <FontAwesomeIcon icon={faTimes} />
+            </button>
+            <button className="btn btn-link my-2 d-none d-md-block" type="button" onClick={() => this.setState({ editingSubscription: !editingSubscription, activeSubscription: false })}>
+              <FontAwesomeIcon icon={faTimes} />
+            </button>
+          </div>
+          <UpdateSubscription
+          updateSubscription={this.handleUpdate}
+          showSubscriptionList={this.showSubscriptionList}
+          toggleLoadingState={this.toggleLoadingState}
+          prevSubscription={editingSubscription}
+          />
+        </div>;
     return (
-      <div>
-        <header>
-          <h1>Hello this is the users dashboard page.</h1>
-          <section className="subscription-list">
-            <SubscriptionsList />
-          </section>
-          {successMessage}
-          <a href={href}>
-            Sign Out!
-            <FontAwesomeIcon icon={faSignOutAlt} />
+      <div className="h-100 d-flex flex-column">
+        <header className="navbar p-2 d-flex justify-content-between align-items-center text-dark border-bottom shadow-sm">
+          <a className="navbar-brand d-flex text-dark" href="#changeThis">
+            <img src={logo} alt="wateringCanIcon" height="60" />
+            <div className="align-self-center d-none d-md-block">Water Your Subs</div>
           </a>
+          <h1 className="display-4 fw-normal">{`${new Date().toDateString()}`}</h1>
```

Comment:
we should modify this a bit to have a comma between the day and year. right now it looks like `Wed March 23 2022` but it should look like `Wed March 23, 2022`